com.github.philip_scott.spice-up.json

```diff
@@ -0,0 +1,95 @@
+            "--socket=session-bus",
+            "--talk-name=ca.desrt.dconf",
+            "--filesystem=xdg-run/dconf",
+            "--filesystem=~/.config/dconf:ro",
+            "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+    "modules": [
+        {
+        "name": "libgee",
+            "build-options": {
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.gnome.org/sources/libgee/0.20/libgee-0.20.0.tar.xz",
+                    "sha256": "21308ba3ed77646dda2e724c0e8d5a2f8d101fb05e078975a532d7887223c2bb"
+                }
+            ]
+        },
+        {
+            "name": "granite",
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_INSTALL_LIBDIR=/app/lib"
+            ],
+            "build-options": {
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+            },
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://launchpad.net/granite/0.4/0.4.1/+download/granite-0.4.1.tar.xz",
+                    "sha256": "8bb411523e6cd5685c928dbfe08893892189cedf9fad5f537aa76182032cf09c"
+                }
+            ]
+        },
+        {
+            "name": "udev",
+            "config-opts": [ "--disable-hwdb", "--disable-logging",
+                            "--disable-introspection", "--disable-keymap", "--disable-mtd_probe"],
+            "cleanup": ["/share", "/include", "/bin", "/sbin"],
+            "sources": [
+            {
+                "type": "archive",
+                "url": "http://kernel.org/pub/linux/utils/kernel/hotplug/udev-175.tar.bz2",
+                "sha256": "4c7937fe5a1521316ea571188745b9a00a9fdf314228cffc53a7ba9e5968b7ab"
+            }
+            ]
+        },
+        {
+            "name": "libevdev",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://www.freedesktop.org/software/libevdev/libevdev-1.5.6.tar.xz",
+                    "sha256": "ecec7e9d66b1d3692f10b3b20aa97fb25e874a784c5552a7b1698091fef5a688"
+                }
+            ]
+        },
+        {
+            "name": "spiceup",
+            "buildsystem": "cmake",
+            "sources": [
+                {
+                    "type": "git",
+                    "url": "https://github.com/Philip-Scott/Spice-up/",
+                    "commit": "f579f4f69c66072333a0cc5bd700f0845aa1d6e1"
```

Comment:
Probably worth using the tarball from the stable release instead: https://github.com/Philip-Scott/Spice-up/releases/tag/1.0.0

Reply:
I looked at that - but isn't that an autogenerated GitHub tarball? We've had problems with them getting purged and regenerated before since that changes the sha256sum. The specified commit should be representing that release.

Reply:
I didn't think it was autogenerated, or at least I wasn't aware that it was the case! Using the commit ID is OK then.

Reply:
They are autogenerated but I have never experienced github changing tarballs behind my back.

Personally I would always go out of our way to avoid git sources as they are just extremely slow compared to a tarball download.

Reply:
Cool - will change it. My irrational prejudice may actually stem from bitbucket, but happy to switch for now.